init_linux.go

```diff
@@ -99,7 +100,12 @@ func finalizeNamespace(config *initConfig) error {
 	if err := utils.CloseExecFrom(3); err != nil {
 		return err
 	}
-	w, err := newCapWhitelist(config.Config.Capabilities)
+
+	capabilities := config.Config.Capabilities
```

Comment:
Ha, ha yes, when I was looking at it before sending PR, I was sure I had done something wrong.